src/GeositeFramework/region.json

```diff
@@ -159,5 +159,14 @@
               }
             ]
         }
-    ]
+    ],
+    "legend": {
+        "layers": [
+            {
+                "serviceUrl": "http://dev.services2.coastalresilience.org:6080/arcgis/rest/services/Gulf_of_Mexico/Florida/MapServer",
+                "layerName": "Bathymetry (m)",
+                "legendType": "scale"
```

Comment:
It's a required field. Our version of jsonschema doesn't support enums, unfortunately, so this will default to "multiple" if the value is not one of the supported legend types.